services/launcher-dev/app/routes.py

```diff
@@ -128,7 +78,7 @@ def compile_and_test():
 
     # We copy the source file from /uploads to a fresh directory in /compilations
     dir_for_compilation = basename
-    path_for_compilations = os.path.join('/compilations', dir_for_compilation)
+    path_for_compilations = os.path.join('/uploads/compilations', dir_for_compilation)
```

Comment:
I is not missing. It is mounted by this line : 

https://github.com/CryptoExperts/wb_contest_submission_server/blob/0f5623aba84f271fa449eac95d4d7c131c12602c/docker-stack-prod.yml#L48